Makefile.nt

```diff
@@ -373,7 +373,7 @@ otherlibs/dynlink/dynlink.cmxa: otherlibs/dynlink/natdynlink.ml
 
 utils/config.ml: utils/config.mlp config/Makefile
 	@rm -f utils/config.ml
-	sed -e "s|%%LIBDIR%%|$(LIBDIR)|" \
+	sed -e "s|%%LIBDIR%%|$(shell echo $(LIBDIR)|sed -e 's/\//\\\\\\\\\\\\\\\\/g')|" \
```

Comment:
Perhaps this fine escape sequence could be preceded by a comment reminding the reader what it does.
Also, it might be clearer if another character (e.g. colon) were used as the sed command delimiter, as then you will remove one backslash.  The same applies below.